decidim-core/app/helpers/decidim/widget_urls_helper.rb

```diff
@@ -2,8 +2,8 @@
 module Decidim
   module WidgetUrlsHelper
     def embed_modal_for(url)
-      js_embed_code = "#{content_tag(:script, '', src: url)}"
-      embed_code = "#{content_tag(:noscript, content_tag(:iframe, '', src: url.gsub(".js", ".html"), frameborder: 0, scrolling: "vertical"))}"
+      js_embed_code = content_tag(:script, "", src: url).to_s
```

Comment:
Yes, it's weird but "content_tag" returns a SafeBuffer, which has the "to_s" method overwritten to return self. In order to test this, visit a participatory process in the development app and try to embed it (on the sidebar). If the code works fine, you'll be able to copy a raw HTML code. If it doesn't, you'll see some weird layout right after the page footer (that's how we checked it)